File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC_CSharp/Networking/dto/ChildDTO.cs
namespace Networking.dto;

[Serializable]
public class ChildDTO
{
    public string Name { get; set; }
    public string CNP { get; set; }

    public ChildDTO(string name, string cnp)
    {
        Name = name;
        CNP = cnp;
    }

    public override string ToString()
    {
        return $"ChildDTO[{Name} {CNP}]";
    }
}

File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC/Problema8SC/Problema8Networking/src/main/java/problema8/network/dto/ChildDTO.java
package problema8.network.dto;

import java.io.Serializable;

public class ChildDTO implements Serializable {
    private String Name;
    private String CNP;

    public ChildDTO(String name, String CNP) {
        this.Name = name;
        this.CNP = CNP;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        this.Name = name;
    }

    public String getCNP() {
        return CNP;
    }

    public void setCNP(String CNP) {
        this.CNP = CNP;
    }

    @Override
    public String toString() {
        return "ChildDTO[" + Name + " " + CNP + "]";
    }
}
